google/cloud/firestore_v1/base_client.py

```diff
@@ -178,13 +178,9 @@ def _emulator_channel(self, transport):
         # "transport.create_channel" to create gRPC channels once google-auth
         # extends it's allowed credentials types.
         if "GrpcAsyncIOTransport" in str(transport.__name__):
-            return grpc.aio.secure_channel(
-                self._emulator_host, self._local_composite_credentials()
-            )
+            return grpc.aio.insecure_channel(self._emulator_host)
```

Comment:
Can we possibly have our cake and eat it too by introducing another environment variable for these custom credentials?